public/.htaccess.dist

```diff
@@ -83,4 +83,10 @@ DirectoryIndex index.php
   </IfModule>
 </IfModule>
 
+# Uncomment if you use folder protection via .htaccess/.htpasswd in accordance to documentation
+# <RequireAny>
+#    Require expr %{THE_REQUEST} =~ m#.*?\s+\/api.*?#
```

Comment:
Is there any reason why you use directly the raw request URI and not the parsed one `Request_URI` ?